translations/README-es.md

```diff
@@ -512,203 +518,202 @@ Puede ver mi código aquí:
-        - [ ] reverse() – Pone al revés toda la lista
-        - [ ] remove_value(value) – Borra el primer elemento en la lista con ese valor
-    - [ ] Doble lista enlazada
-        - [Description (video)](https://www.coursera.org/learn/data-structures/lecture/jpGKD/doubly-linked-lists)
-        - No es necesario
-- ### Pila
-    - [ ] [Stacks (video)](https://www.coursera.org/learn/data-structures/lecture/UdKzQ/stacks)
-    - [ ] [Using Stacks Last-In First-Out (video)](https://archive.org/details/0102WhatYouShouldKnow/05_01-usingStacksForLast-inFirst-out.mp4)
+        - [ ] size() – Regresa el número de elementos en la lista.
+        - [ ] empty() – Un bool que regresa verdadero si está vacío.
+        - [ ] value_at(index) – Regresa el valor del el n-avo elemento (usando 0 como primero).
+        - [ ] push_front(value) – Añade un elemento al frente de la lista.
+        - [ ] pop_front() – Elimina el elemento frontal y regresa su valor.
+        - [ ] push_back(value) – Añade un elemento al final.
+        - [ ] pop_back() – Elimina el último elemento y devuelve su valor.
+        - [ ] front() – Obtiene el valor del elemento frontal.
+        - [ ] back() – Obtiene el valor del último elemento.
+        - [ ] insert(index, value) – Inserta valor en el índice, por lo que el valor actual en ese índice es apuntado al nuevo elemento en el índice.
+        - [ ] erase(index) – Borra el nodo del índice dado.
+        - [ ] value_n_from_end(n) – Regresa el valor del nodo en la n-ava posición a partir del final de la lista.
+        - [ ] reverse() – Pone al revés toda la lista.
+        - [ ] remove_value(value) – Borra el primer elemento en la lista con ese valor.
+    - [ ] Doble lista enlazada:
+        - [Description (video)](https://www.coursera.org/learn/data-structures/lecture/jpGKD/doubly-linked-lists).
+        - No es necesario.
+- ### Pila:
+    - [ ] [Stacks (video)](https://www.coursera.org/learn/data-structures/lecture/UdKzQ/stacks).
+    - [ ] [Using Stacks Last-In First-Out (video)](https://archive.org/details/0102WhatYouShouldKnow/05_01-usingStacksForLast-inFirst-out.mp4).
     - [ ] No implementaré. Implementando con arreglos es trivial.
 
 - ### Cola o fila
-    - [ ] [Using Queues First-In First-Out(video)](https://archive.org/details/0102WhatYouShouldKnow/05_03-usingQueuesForFirst-inFirst-out.mp4)
-    - [ ] [Queue (video)](https://www.coursera.org/learn/data-structures/lecture/EShpq/queue)
-    - [ ] [Circular buffer/FIFO](https://en.wikipedia.org/wiki/Circular_buffer)
-    - [ ] [Priority Queues (video)](https://archive.org/details/0102WhatYouShouldKnow/05_04-priorityQueuesAndDeques.mp4)
+    - [ ] [Using Queues First-In First-Out(video)](https://archive.org/details/0102WhatYouShouldKnow/05_03-usingQueuesForFirst-inFirst-out.mp4).
+    - [ ] [Queue (video)](https://www.coursera.org/learn/data-structures/lecture/EShpq/queue).

```

Comment:
There are a lot of lines like this where only a period was added. Please remove the extra period from those lines to make it easier to see the differences.

Reply:
Just removed most of the periods added... I just reviewed the punctuation rules and noticed that in listings in which long sentences are not included, periods should not be added at the end. 

Let me know what you think.